iam-password-policy/change-password-policy.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+
+from copy import Error
+import boto3
+from botocore.exceptions import ClientError
+import os
+import logging
+from botocore.session import Session
+
+
+
+
+
+def main(args, logger):
+    '''Executes the Primary Logic of the Fast Fix'''
+
+    # If they specify a profile use it. Otherwise do the normal thing
+    if args.profile:
+        session = boto3.Session(profile_name=args.profile)
+    else:
+        session = boto3.Session()
+
+    try:
+        response = setPasswordPolicy(args, client= session.client(service_name='iam'))
+        logging.debug("Response:"+str(response))
+        logging.info("Response: Successfuly updated password policy")
+    except Error as err:
+        logging.error("Authentication Failed, Network Connectivity is an issue, or Something else went wrong. Use --debug for more information.")
+        logging.debug("error:"+str(err))
+        logging.debug()
+
+
+def setPasswordPolicy( args ,client= None,):
+    #Excute account password policy change
+   
+    currentAccountPolicy = client.get_account_password_policy()
+    changes_required = False
+
+    strong_stance = {
+        "MinimumPasswordLength": 14,
+        "RequireSymbols": True,
+        "RequireNumbers": True,
+        "RequireUppercaseCharacters": True,
+        "RequireLowercaseCharacters": True,
+        "AllowUsersToChangePassword": True,
+        "MaxPasswordAge": 180,
+        "PasswordReusePrevention": 15,
+        "HardExpiry": False }
+    if not args.auto:
+        inputPolicy= {
+            "MinimumPasswordLength": args.MinimumPasswordLength,
+            "RequireSymbols": args.RequireSymbols,
+            "RequireNumbers": args.RequireNumbers,
```

Comment:
How was this tested? I get an automatic failure when running with no arguments
```./change-password-policy.py --debug
Traceback (most recent call last):
  File "./change-password-policy.py", line 170, in <module>
    main(args, logger)
  File "./change-password-policy.py", line 24, in main
    response = setPasswordPolicy(args, client= session.client(service_name='iam'))
  File "./change-password-policy.py", line 52, in setPasswordPolicy
    "RequireNumbers": args.RequireNumbers,
AttributeError: 'Namespace' object has no attribute 'RequireNumbers'
```

Reply:
Require Numbers wasnt instantiated.

Thats my secret Captain. This was written from documentation only.